azurerm/internal/services/containers/resource_arm_kubernetes_cluster.go

```diff
@@ -1427,15 +1553,42 @@ func flattenKubernetesClusterNetworkProfile(profile *containerservice.NetworkPro
 		podCidr = *profile.PodCidr
 	}
 
+	lbProfiles := make([]interface{}, 0)
+	if profile.LoadBalancerProfile != nil {
+		lb := make(map[string]interface{})
+
+		if profile.LoadBalancerProfile.ManagedOutboundIPs != nil {
+			if profile.LoadBalancerProfile.ManagedOutboundIPs.Count != nil {
+				lb["managed_outbound_ips"] = profile.LoadBalancerProfile.ManagedOutboundIPs.Count
+			}
+		}
```

Comment:
and then this could become
```
		if ips := lbp.ManagedOutboundIPs != nil {
			if count := ips.Count != nil {
				lb["managed_outbound_ips"] = count
			}
		}
```